merge.go

```diff
@@ -487,7 +487,12 @@ func mergeTermFreqNormLocs(fieldsMap map[string]uint16, term []byte, postItr *Po
 		newRoaring.Add(uint32(hitNewDocNum))
 
 		nextFreq := next.Frequency()
-		nextNorm := uint64(math.Float32bits(float32(next.Norm())))
+		var nextNorm uint64
+		if pi, ok := next.(postingsAdv); ok {
```

Comment:
sorry I didn't notice this during the discussion.  why do we even need the private interface?  couldn't we simply type assert to the implementation struct?

`next.(*Posting)`